README.rst

```diff
@@ -32,6 +32,14 @@ v0.3.0 (2020-09-03)
 * rewrote RemoteDispatcher to use BlueskyConsumer
 * changed default serialization method to MessagePack
 
+upcoming changes
```

Comment:
Maybe add this to release notes?

Reply:
That was my weak attempt to "draft" the next version of release notes. We don't have a separate file for release notes, they are just in the readme. Do you think we should add a separate release notes file?

Reply:
We can add it later?